<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNavListViewForm</class>
 <widget class="QWidget" name="frmNavListViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="NavListView" name="listView">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page1">
      <widget class="QLabel" name="label_1">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>160</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>结账功能</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>141</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>已购商品：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>141</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>会员名：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_vip">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>230</y>
         <width>290</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>430</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>总价：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_total">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>410</y>
         <width>290</width>
         <height>80</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>290</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_cashpay">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>540</y>
         <width>130</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>现金结账</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_cardpay">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>660</y>
         <width>130</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>会员卡结账</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>540</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>实收：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_income">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>540</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>540</y>
         <width>61</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>找零：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_outgoing">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>540</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>660</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>会员卡金额：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_account">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>660</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>660</y>
         <width>61</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>余额：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_remain">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>660</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>430</y>
         <width>130</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>计 算</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>330</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>会员手机号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_vip_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>330</y>
         <width>290</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textEdit_goods">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>120</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>130</y>
         <width>130</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>打开扫码枪</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page2">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>200</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>添加新会员</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>新会员账号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>会员手机号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>490</y>
         <width>140</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>首次充值：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_newvipName">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>130</y>
         <width>300</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_newvipPhone">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>250</y>
         <width>300</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_newvipRegister">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>610</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>注  册</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>370</y>
         <width>171</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>确认手机号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_newvipContirm">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>370</y>
         <width>300</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_newvipFirst">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>490</y>
         <width>300</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page3">
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>211</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>老会员管理</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>130</y>
         <width>131</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>会员账号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>350</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>修改手机号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>590</y>
         <width>131</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>会员充值：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_oldvipName">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>130</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_oldvipNewPhone">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>350</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_oldvipCOnfirm">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>470</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_changeVipphone">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>470</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>修 改</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_addVipcount">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>590</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>充 值</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>240</y>
         <width>131</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>原手机号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_oldvipPhone">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>240</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>470</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确认手机号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_oldvipAddcount">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>590</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page4">
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>240</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>退/换货管理</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>261</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>退/换货商品及数量：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_returnGoods">
       <property name="geometry">
        <rect>
         <x>429</x>
         <y>150</y>
         <width>311</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>340</y>
         <width>171</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>退/换货原因：</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_returnReason">
       <property name="geometry">
        <rect>
         <x>429</x>
         <y>325</y>
         <width>311</width>
         <height>70</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>500</y>
         <width>161</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>退款金额：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_returnTotal">
       <property name="geometry">
        <rect>
         <x>429</x>
         <y>500</y>
         <width>311</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pB_return">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>620</y>
         <width>160</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>确认退换货</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>301</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>200</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>主界面</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>124</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>470</y>
         <width>151</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>音乐播放器</string>
       </property>
      </widget>
      <widget class="QListView" name="listView_2">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>520</y>
         <width>731</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_personInfo">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>190</y>
         <width>281</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>130</y>
         <width>171</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>个人本月绩效</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>128</y>
         <width>93</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavListView</class>
   <extends>QListView</extends>
   <header>navlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
